### PR TITLE
Add Stripe invoices and payment methods for projects to console

### PR DESCRIPTION
**Pull request by rikysya**
What: add invoices to console

Why:

Thanks for submitting a PR!

## Code Review Checklist (to be filled out by reviewer)
 - [ ] Does the PR describe what changes are being made?
 - [ ] Does the PR describe why the changes are being made?
 - [ ] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [ ] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [ ] Could the PR be broken into smaller PRs?
 - [ ] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [ ] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [ ] Does any documentation need updating?

### CODE REVIEW

File: internal/payments/service.go [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+// Created invoice has AutoAdvance property set to true, so it will be finalized
+// (no further editing) and attempted to be paid in 1 hour after creation
+func (s *StripeService) CreateProjectInvoice(ctx context.Context, params CreateProjectInvoiceParams) (*stripe.Invoice, error) {
+	// create line items
+	_, err := s.client.InvoiceItems.New(&stripe.InvoiceItemParams{
+		Customer:    stripe.String(params.CustomerID),
+		Description: stripe.String("Storage"),
+		Quantity:    stripe.Int64(int64(params.Storage)),
+		UnitAmount:  stripe.Int64(100),
+		Currency:    stripe.String(string(stripe.CurrencyUSD)),
```
Comment (VinozzZ) [inline internal/payments/service.go]: Does this mean that we will always use `USD` as the unit for our billing and stripe will handle the currency conversion for us?

Comment (VinozzZ) [reply internal/payments/service.go]: What the impact would be for ppl who are not in the US?

Comment (rikysya) [reply internal/payments/service.go]: Rule like this currency, pricing etc are not defined yet. Additionally for the first few weeks stripe will be operating in test mode, with predefined cards, which will be attached and charged automatically

File: satellite/console/database.go [hunk 2]
```diff
@@ -27,6 +27,12 @@ type DB interface {
 	ResetPasswordTokens() ResetPasswordTokens
 	// UsageRollups is a getter for UsageRollups repository
 	UsageRollups() UsageRollups
+	// UserPaymentInfos is a getter for UserPaymentInfos
+	UserPaymentInfos() UserPaymentInfos
+	// ProjectPaymentInfos is a getter for ProjectPaymentInfos
+	ProjectPaymentInfos() ProjectPaymentInfos
+	// ProjectInvoiceStamps is a getter for ProjectPaymentInfos
```
Comment (VinozzZ) [inline satellite/console/database.go]: There's a typo here. Should `ProjectInvoiceStamps is a getter for ProjectPaymentInfos` be `ProjectInvoiceStamps is a getter for ProjectInvoiceStamps`?

Comment (rikysya) [reply satellite/console/database.go]: yup, you are right

**Comment by rikysya** [comment]
outdated

**Pull request closed without merging**